### PR TITLE
Add support for 2.6 Intree Multitenancy Support

### PR DESCRIPTION
**Pull request by lpabon**

### CODE REVIEW

File: content/cloud-references/security/kubernetes/shared-secret-model-multitenancy/storageclass.md [hunk 1]
```diff
@@ -4,6 +4,27 @@ keywords: storageclass, csi, security, authorization
 weight: 40
 ---
 
+# StorageClass for non-CSI
+
+The following StorageClass enables your tenants to create volumes
```
Comment (nathan-px) [inline content/cloud-references/security/kubernetes/shared-secret-model-multitenancy/storageclass.md:9]:
```suggestion
The following StorageClass allows your tenants to create volumes
```

File: content/cloud-references/security/kubernetes/shared-secret-model-multitenancy/storageclass.md [hunk 2]
```diff
@@ -4,6 +4,27 @@ keywords: storageclass, csi, security, authorization
...
+using their token stored in a secret in their namespace.
```
Comment (nathan-px) [inline content/cloud-references/security/kubernetes/shared-secret-model-multitenancy/storageclass.md:10]:
```suggestion
using their token stored in a secret in their namespace:
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/manage.md [hunk 3]
```diff
@@ -8,47 +8,10 @@ series: k8s-op-maintain-auth
...
-    openstorage.io/auth-secret-name: px-secret
-    openstorage.io/auth-secret-namespace: default
-spec:
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: 2Gi
-```
+For more information please see [Securing your Portworx system](/cloud-references/security/).
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/manage.md:14]:
```suggestion
For more information, please see [Securing your Portworx system](/cloud-references/security/).
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/manage.md [hunk 4]
```diff
@@ -8,47 +8,10 @@ series: k8s-op-maintain-auth
 
 ## Creating volumes
 Portwox authorization provides a method of protection for creating volumes
-through Kubernetes. Users must provide a token when requesting volumes. These
-tokens must be saved in a Secret, normally in the same namespace as the PVC.
+through Kubernetes. PVCs must refer to a StorageClass which point to the
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/manage.md:11]:
```suggestion
through Kubernetes. PVCs must refer to a StorageClass which points to the
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/enable.md [hunk 5]
```diff
@@ -6,7 +6,12 @@ weight: 100
 series: k8s-op-maintain-auth
 ---
 
-Before proceeding with this installation, please review the [Security](/concepts/authorization) model used by Portworx.
+Before proceeding with this document, please review the [Security](/concepts/authorization) model used by Portworx.
+
+{{<info>}}
+For a step by step setup of guide of how to enable Portworx authorization, please see
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/enable.md:12]:
```suggestion
**NOTE:** For a step by step setup guide showing how to enable Portworx authorization, please see
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/_index.md [hunk 6]
```diff
@@ -7,7 +7,12 @@ series: k8s-op-maintain
 keywords: authorization, security, kubernetes, k8s
 ---
 
-This section describes how to enable and manage Portworx authorization in
+This is a reference section describing how to manage Portworx authorization in
 Kubernetes
 
+{{<info>}}
+For a step by step setup of guide of how to enable Portworx authorization, please see
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/_index.md:14]:
```suggestion
**NOTE:** For a step-by-step setup guide of how to enable Portworx authorization, please see
```

File: content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/_index.md [hunk 7]
```diff
@@ -7,7 +7,12 @@ series: k8s-op-maintain
 keywords: authorization, security, kubernetes, k8s
 ---
 
-This section describes how to enable and manage Portworx authorization in
+This is a reference section describing how to manage Portworx authorization in
```
Comment (nathan-px) [inline content/portworx-install-with-kubernetes/operate-and-maintain-on-kubernetes/authorization/_index.md:10]:
```suggestion
This reference section describes how you can manage Portworx authorization in
```

**Comment by lpabon** [comment]
No longer necessary. We do not support this mode anylonger. It was added then removed from 2.6

**Pull request closed without merging**